faucet-server/Sources/App/Services/CKBService.swift

```diff
@@ -50,13 +50,13 @@ class CKBService {
 
     static func generatePrivateKey() -> String {
         var data = Data(repeating: 0, count: 32)
-        if #available(OSX 10.7, *) {
-            data.withUnsafeMutableBytes({ _ = SecRandomCopyBytes(kSecRandomDefault, 32, $0.baseAddress! ) })
-        } else {
-            for _ in 0..<32 {
-                data.append(UInt8.random(in: UInt8.min...UInt8.max))
-            }
+        #if os(OSX)
+        data.withUnsafeMutableBytes({ _ = SecRandomCopyBytes(kSecRandomDefault, 32, $0.baseAddress! ) })
+        #else
+        for _ in 0..<32 {
```

Comment:
This creates 64 bytes data with the first half all zeros.

Reply:
@ashchan  https://github.com/nervosnetwork/ckb-testnet-faucet/pull/45